source/extensions/filters/http/ext_proc/ext_proc.cc

```diff
@@ -1605,10 +1706,16 @@ void Filter::onReceiveMessage(std::unique_ptr<ProcessingResponse>&& r) {
     processing_status = encoding_state_.handleHeadersResponse(response->response_headers());
     break;
   case ProcessingResponse::ResponseCase::kRequestBody:
+    if (response->has_request_body()) {
```

Comment:
qq; why we need this `response->has_request_body` . Do we receive empty response when response case is set to kRequestBody?